api/search.go

```diff
@@ -152,6 +153,21 @@ func NewSearch(client *gosip.SPClient, endpoint string, config *RequestConfig) *
 	}
 }
 
+func (search *Search) GetQuery(query *SearchQuery) (SearchResp, error) {
```

Comment:
Will it bling any value while providing the same exactly functionality as search.PostQuery? What are you trying to solve?